gbpservice/nfp/base_configurator/api/v1/controllers/controller.py

```diff
@@ -56,27 +50,16 @@ def __init__(self, method_name):
         self.vm_port = '8080'
         self.max_retries = 12
 
-    def _push_notification(self, context, request_info, result, config_data):
+    def _push_notification(self, context, result, config_data, service_type):
         global notifications
         resource = config_data['resource']
-        receiver = ''
-        for key in self.resource_map.keys():
-            if resource in key:
-                receiver = self.resource_map[key]
-
-        response = {
-            'receiver': receiver,
-            'resource': resource,
-            'method': 'network_function_device_notification',
-            'kwargs': [
-                {
-                    'context': context,
-                    'resource': resource,
-                    'request_info': request_info,
-                    'result': result
-                }
-            ]
-        }
+
+        response = {'info': {'service_type': service_type,
+                             'context': context},
+                    'notification': [{
+                          'resource': resource,
+                          'data': {'status_code': UNHANDLED}}]
```

Comment:
Is base mode with VM working with this change? We seem to return unhandled for all the notifications.